dpnp/tests/test_mathematical.py

```diff
@@ -1415,9 +1426,20 @@ def test_basic(self, mant_dt, exp_dt):
         exp = numpy.array(3, dtype=exp_dt)
         imant, iexp = dpnp.array(mant), dpnp.array(exp)
 
-        result = dpnp.ldexp(imant, iexp)
-        expected = numpy.ldexp(mant, exp)
-        assert_almost_equal(result, expected)
+        if dpnp.issubdtype(exp_dt, dpnp.uint64):
+            assert_raises(ValueError, dpnp.ldexp, imant, iexp)
+            assert_raises(TypeError, numpy.ldexp, mant, exp)
+        elif dpnp.issubdtype(exp_dt, dpnp.uint32):
+            # For this special case, NumPy raises an error on Windows
```

Comment:
Do that depends on numpy version? Is the issue visible with numpy 2.2?

Reply:
It is resolved in numpy 2.0 and afterwards. Updated the code to consider this fact.